docs/user-guide/consensus-clients.rst

```diff
@@ -434,13 +434,13 @@ receiving tips and set the fee recipient flag:
 
 .. prompt:: bash $
 
-  sudo sed -i 's/changeme/$YOUR_ETH_ADDRESS' /etc/ethereum/nimbus-beacon.conf
+  sudo sed -i 's/changeme/$YOUR_ETH_ADDRESS' /etc/ethereum/nimbus-validator.conf
```

Comment:
I personally set --suggested-fee-recipient on both beacon and validator configuration, just in case :)